tvloader/parser2v1/parse_package.go

```diff
@@ -20,6 +20,10 @@ func (parser *tvParser2_1) parsePairFromPackage2_1(tag string, value string) err
 	case "PackageName":
 		// if package already has a name, create and go on to a new package
 		if parser.pkg == nil || parser.pkg.PackageName != "" {
+			// check if the previous package contained a spdxId or not
+			if parser.pkg != nil && parser.pkg.PackageSPDXIdentifier == nullSpdxElementId2_1 {
+				return fmt.Errorf("invalid package without a package SPDX identifier")
```

Comment:
Similar to the File error messages, can you tell the user in this error message what is the Package's PackageName, so that they can tell which one is incorrect?